Azure/sonic-buildimage
# Fix read fan_fault=1 and send log

Fix read fan_fault=1 and send log when fan insert sometimes
**- What I did**
Fix read fan_fault=1 and send log when fan insert sometimes
**- How I did it**
Add check fan_fault  when fan_status=fan_fault . If fan_fault become to zero, log "FAN-N becomes operational"
**- How to verify it**
Run monitor_fan.py. and check log as below,
 root        : INFO     FAN-6 present is detected
root        : WARNING  Alarm for FAN-6 failed is detected
root        : INFO     FAN-6 becomes operational
**- Description for the change log**



**- A picture of a cute animal (not mandatory but encouraged)**
